x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/serialization/policy_serialization.test.ts

```diff
@@ -91,6 +92,78 @@ describe('<EditPolicy /> serialization', () => {
         },
       });
     });
+
+    it('default policy (only policy name input) on enterprise license', async () => {
```

Comment:
was this test copied from somewhere else?

Reply:
I created those 2 tests because I noticed a bug with frozen phase: on enterprise license, a policy can be saved with only name input, but on basic license, the page crushed. So I wanted to test a minimal input policy for both licenses.